src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,491 @@
+package strawman
+package collection.mutable
+
+import scala._
+import scala.collection.{generic, mutable}
```

Comment:
Currently it seems that the tests don’t pass with 2.12: https://travis-ci.org/scala/collection-strawman/builds/263155071#L974